tests/boot/boot_from_pxe.pm

```diff
@@ -21,27 +21,60 @@ use testapi;
 use bootloader_setup qw(bootmenu_default_params specific_bootmenu_params);
 use registration 'registration_bootloader_cmdline';
 use utils 'type_string_slow';
+use backend::ipmi 'ipmitool';
+
+# Variables are present in vars.json
+# IPMI_HOSTNAME, IPMI_USER, IPMI_PASSWORD
+my $ipmi_host = {
+    hostname => get_required_var('IPMI_HOSTNAME'),
+    username => get_required_var('IPMI_USER'),
+    userpass => get_required_var('IPMI_PASSWORD'),
+};
+
+my $cmd_prefix = "ipmitool -I lanplus -H $ipmi_host->{hostname} -U $ipmi_host->{username} -P $ipmi_host->{userpass} -v ";
```

Comment:
why not use `backend::impi_cmdline` instead?

Reply:
I was playing with the interface and afterward, I did not feel like I need to introduce another dependency.

Reply:
I'd still prefer to not reinvent the wheel, however ipmi_cmdline does not check that all of the variables are there (It just goes through bmwqemu directly) so, I'm neutral :)

Reply:
then we might consider changing that in `ipmi_cmdline`, of course doesn't mean right now =)

Reply:
Maybe ipmi backend can be refactored as whole one day in the future. :)